packages/wdio-cli/src/utils.ts

```diff
@@ -11,6 +11,10 @@ import type { Options, Capabilities, Services } from '@wdio/types'
 
 import { ReplCommandArguments, Questionnair, SupportedPackage, OnCompleteResult, ParsedAnswers } from './types'
 import { EXCLUSIVE_SERVICES, ANDROID_CONFIG, IOS_CONFIG, QUESTIONNAIRE } from './constants'
+import { ConfigParser } from '@wdio/config'
+import { DesiredCapabilities, MultiRemoteCapabilities, W3CCapabilities } from '@wdio/types/build/Capabilities'
+import pickBy from 'lodash.pickby'
+import _ from 'lodash'
```

Comment:
Let's not import all of `lodash` , it is a massive library and many of it isn't used here. Better add a specific method as dependency and import it. In this case I would recommend to just use basic JS to check if `requiredW3CCaps` exists.